### ISSUE TITLE
Connecting a LSTM to ConvLSTM2D's output

### ISSUE THREAD

**Issue by korrahman** [keras-team/keras#11713]
Hey there!
I need to connect output of ConvLSTM2D to a LSTM!
The output of ConvLSTM2D is (samples, time, output_row, output_col, filters) when "return_sequences" is 'True'.
I’m confused here how this 5-D input can be feed to a LSTM! As you know if I want to reshape, 'samples' and 'time' should not be changed and 'output_row' 'output_col' 'filters'  should be change to a vector(1-D).
Am I right? How to do that?
I will be thankful if any help!

>  #creating our model
> myInput = layers.Input(shape=(3,30,3,1))
> conv1 = layers.TimeDistributed(layers.Conv2D(16, 3, activation='relu', padding='same', strides=2, data_format = 'channels_first'))(myInput)
> BN1 = layers.TimeDistributed(layers.BatchNormalization())(conv1)
> conv2 = layers.TimeDistributed(layers.Conv2D(16, 3, activation='relu', padding='same', strides=2, data_format = 'channels_first'))(BN1)
> BN2 = layers.TimeDistributed(BatchNormalization())(conv2)
> 
> convlstm1 = layers.ConvLSTM2D(filters=3, kernel_size=3, padding='same',data_format='channels_last', return_sequences=True)(BN2)
> res1 = keras.layers.Add()([convlstm1, BN2])
> 
> convlstm2 = layers.ConvLSTM2D(filters=3, kernel_size=3, padding='same', return_sequences=True)(res1)
> res2 = keras.layers.Add()([convlstm2, res1])
> 
> convlstm3 = layers.ConvLSTM2D(filters=3, kernel_size=3, padding='same', return_sequences=True)(res2)
> res3 = keras.layers.Add()([convlstm3, res2])
![capture](https://user-images.githubusercontent.com/45286655/48940085-affc9b80-ef2b-11e8-8660-6e94f4a6a496.PNG)

**Comment by ymodak** [comment]
Automatically closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!
